docs/support/add_iss_provision.md

```diff
@@ -0,0 +1,103 @@
+# Add an ISS Provision
+
+An Independent Special School (ISS) Provision allows a `LocalAuthority`, a type of `ResponsibleBody`, 
+to order devices on behalf of students in independent special schools. These are typically pupils
+whose fees are being paid for them in order to attend an independent special school which can
+better cater to their needs than a state school.
+
+An ISS Provisioned `LaFundedPlace`, a type of `School`, will always have the same name 
+"State-funded pupils in independent special schools and alternative provision". In our system it's 
+modelled as a `School` but really it isn't: it's just a representation of a number of pupils at 
+different ISSs overseen by a single `LocalAuthority`. Thereby the `name` attribute has to be 
+generic. 
+
```

Comment:
Maybe add: "This is why a `LocalAuthority` only `has_one iss_provision` and/or `has_one scl_provision`.

Reply:
Otherwise LGTM.